app/models/fixed_asset.rb

```diff
@@ -237,6 +238,10 @@ def opened_financial_year?
     FinancialYear.on(started_on)&.opened?
   end
 
+  def financial_year_in_closure_preparation?
```

Comment:
Can this be more explicit? Like `started_during_closure?` or something, that makes clear how the financial year is found?